packages/core/src/fhir-to-cda/cda-templates/commons.ts

```diff
@@ -257,11 +262,13 @@ export function formatDateToHumanReadableFormat(
     }
     return date.utc().format("MM/DD/YYYY h:mm A");
   }
-
-  // if (dateString.includes("T")) return date.utc().format("MM/DD/YYYY h:mm A");
   return date.utc().format("MM/DD/YYYY");
 }
-// see https://build.fhir.org/ig/HL7/CDA-core-sd/StructureDefinition-ST.html
+
+/**
+ * ST stands for SimpleText
+ * See https://build.fhir.org/ig/HL7/CDA-core-sd/StructureDefinition-ST.html for more details
```

Comment:
There's a TSDoc annotation for "see", it's `@see` - https://tsdoc.org/pages/tags/see/

Reply:
Done